Tella/Components/Reports/BaseReportsViewModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  BaseReportsViewModel.swift
+//  Tella
+//
+//  Created by gus valbuena on 6/27/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+protocol ReportsViewModelProtocol: ObservableObject {
+    var selectedCell: Page { get set }
+}
```

Comment:
Same remark here for the `selectedCell` naming

Reply:
we end up not using this protocol so I'll just remove this file